src/backend/dx12/src/lib.rs

```diff
@@ -1286,6 +1286,21 @@ impl hal::Instance<Backend> for Instance {
                 _ => unreachable!(),
             } as _;
 
+            let mut tiled_resource_features = Features::empty();
+            if features.TiledResourcesTier >= d3d12::D3D12_TILED_RESOURCES_TIER_1 {
+                tiled_resource_features |= Features::SPARSE_BINDING;
+                tiled_resource_features |= Features::SPARSE_RESIDENCY_IMAGE_2D;
+                tiled_resource_features |= Features::SPARSE_RESIDENCY_BUFFER;
+                tiled_resource_features |= Features::SPARSE_RESIDENCY_ALIASED;
+                tiled_resource_features |= Features::SPARSE_RESIDENCY_2_SAMPLES;
+                tiled_resource_features |= Features::SPARSE_RESIDENCY_4_SAMPLES;
+                tiled_resource_features |= Features::SPARSE_RESIDENCY_8_SAMPLES;
+                tiled_resource_features |= Features::SPARSE_RESIDENCY_16_SAMPLES;
+            }
+            if features.TiledResourcesTier >= d3d12::D3D12_TILED_RESOURCES_TIER_3 {
```

Comment:
we have stuff enabled by TIER_1, then TIER_3. There was nothing enabled in TIER_2?

Reply:
Tier 2 allows for shader instructions clamping level of detail, defined behaviour for reading and writing to null mapped tiles and some mip packing definitions too, but nothing that is under vulkan features.

Reply:
There's also an undocummented tier 4 which isn't part of the `winapi` crate.